cpu/stm32_common/cpu_init.c

```diff
@@ -49,6 +50,44 @@ void cpu_init(void)
     periph_clk_en(APB1, BIT_APB_PWREN);
     /* initialize the system clock as configured in the periph_conf.h */
     stmclk_init_sysclk();
+
+#if defined(CPU_FAM_STM32L1)
+    uint32_t ahb_gpio_clocks;
+    GPIO_TypeDef *port;
+
+    /* enable GPIO clock and save GPIO clock configuration */
+    ahb_gpio_clocks = RCC->AHBENR & 0xFF;
```

Comment:
Is there a define for `0xff` in the cmsis header file ? Maybe use it instead of hard coded values.

Reply:
There is not definition in the cmsis header file, I also checked the newer versions and there isn't one.